storage/client/file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java

```diff
@@ -74,9 +74,7 @@ public AzureFileStorageImpl build() {
             this.pipeline = RestProxy.createDefaultPipeline();
         }
         AzureFileStorageImpl client = new AzureFileStorageImpl(pipeline);
-        if (this.version != null) {
```

Comment:
Does the version always get set from a static conig class ?

Reply:
This is auto-gen class. I have the temp fix to pass the spotbugs. Loop in @jianghaolu for further discussion.

Reply:
Let's track this, so it doesn't cause issues later.